### ISSUE TITLE
Unable to use headless browser(PhantomJS) from VBA 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#108]
```
Operating system : Windows 7 32-bit
.Net Framework version :
Office Version : 2010
SeleniumWrapper version : SeleniumWrapperSetup-1.0.20.0

What is your issue ?

1. I would like to thanks you for the great Selenium VBA Wrapper, I am using it 
on Firefox and Chrome browsers without any issues, still in ways to explorer on 
IE.
   I am very excited to try and test it on phantomJS but stuck-up with issue mention below, may be I am not following right way to use it.
   Could you please provide an example of PhantomJS using it from VBA.

2. Unable to use headless browser(PhantomJS) VBA , Could you please correct 
where I am wrong in this case.

    From https://code.google.com/p/selenium-vba/ found that 

    There are 4 browsers that can be controlled locally that need to be installed to work with:

    Firefox : "firefox"
    Chrome : "chrome"
    InternetExplorer : "ie"
    InternetExplorer 64 bits : "ie64"
    Safari : "safari"
    There is also one headless browser(PhantomJS) included in the installation which behave like a browser but without a window to interect with:
    PhantomJS : "phantomjs"

    When I tried to use PhantomJS from Excel VBA IDE as shown in the attached screen shot,

    "
        Public Sub TC_Browsers()
        Dim driver As New SeleniumWrapper.WebDriver

        driver.Start "phantomjs", "http://news.yahoo.com"
        driver.Open "/science"

        End Sub

    "

    Getting error " Run-time error '-2147467261(80004003)': 
    arguments must not be null
    Parameter name: arguments.

```

Original issue reported on code.google.com by `shukoor...@gmail.com` on 14 Oct 2014 at 3:50

Attachments:
- [PhantomJS.png](https://storage.googleapis.com/google-code-attachments/selenium-vba/issue-108/comment-0/PhantomJS.png)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.0.21.1
```

Original comment by `florentbr` on 14 Oct 2014 at 5:03
- Changed state: **Fixed**
- Added labels: **Type-Defect**
- Removed labels: **Type-Support**

**Comment by GoogleCodeExporter** [comment]
```
Thanks you very much for the quick response, it really worked and resolved the 
issue, I was facing. Your help was really appreciated.

Thanks again, 
```

Original comment by `shukoor...@gmail.com` on 14 Oct 2014 at 6:21
